query/stdlib/influxdata/influxdb/source.go

```diff
@@ -45,7 +45,14 @@ type Source struct {
 func (s *Source) Run(ctx context.Context) {
 	labelValues := s.m.getLabelValues(ctx, s.orgID, s.op)
 	start := time.Now()
-	err := s.runner.run(ctx)
+	var err error
+	if flux.IsExperimentalTracingEnabled() {
```

Comment:
what is the motivation behind making this a global? feels like it should be passed in DI fashion